test/data/basic-shared-project.json

```diff
@@ -0,0 +1,134 @@
+{
+    "jobs": {
+        "main": {
+            "fc6ca4930ee0db5510d8a709b689b29e106586a9": {
+                "image": "node:4",
+                "steps": {
+                    "install": "npm install",
+                    "test": "npm test",
+                    "publish": "npm publish"
+                },
+                "environment": {
+                    "FOO": "bar",
+                    "BAR": "foo",
+                    "DIRECTORY": "c"
+                }
+            },
+            "b5ee24cb8f30df1fd6ae6f4f2d979373fba1f786": {
+                "image": "node:4",
+                "steps": {
+                    "install": "npm install",
+                    "test": "npm test",
+                    "publish": "npm publish"
+                },
+                "environment": {
+                    "FOO": "bar",
+                    "BAR": "foo",
+                    "DIRECTORY": "d"
+                }
+            }
+        },
+        "foobar": {
+            "17147ea3ec8f05bab9c25aa89cd30a7b36b92356": {
+                "image": "node:4",
+                "steps": {
+                    "hostname": "hostname"
+                },
+                "environment": {
+                    "FOO": "ban",
+                    "BAZ": "foo",
+                    "VERSION": 1,
+                    "DIRECTORY": "a"
+                }
+            },
+            "2c10506487da154163121b1b9be0c69d0ccb01cd": {
+                "image": "node:4",
+                "steps": {
+                    "hostname": "hostname"
+                },
+                "environment": {
+                    "FOO": "ban",
+                    "BAZ": "foo",
+                    "VERSION": 1,
+                    "DIRECTORY": "b"
+                }
+            },
+            "dd4e41743cb881768e87641913f67876ce753d69": {
+                "image": "node:4",
+                "steps": {
+                    "hostname": "hostname"
+                },
+                "environment": {
+                    "FOO": "ban",
+                    "BAZ": "foo",
+                    "VERSION": 2,
+                    "DIRECTORY": "a"
+                }
+            },
+            "5a2c7ee6a7233a6ce8b001d55e5404266b55c3d7": {
+                "image": "node:4",
+                "steps": {
+                    "hostname": "hostname"
+                },
+                "environment": {
+                    "FOO": "ban",
+                    "BAZ": "foo",
+                    "VERSION": 2,
+                    "DIRECTORY": "b"
+                }
+            },
+            "cb25c7f5969269c47d0b1206463b98cd1bf2dc61": {
+                "image": "node:4",
+                "steps": {
+                    "hostname": "hostname"
+                },
+                "environment": {
+                    "FOO": "ban",
+                    "BAZ": "foo",
+                    "DIRECTORY": "a",
```

Comment:
Is this environment any different than the `17147ea3ec8f05bab9c25aa89cd30a7b36b92356` environment?